lib/class-wp-rest-media-search-handler-gutenberg.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * REST API: WP_REST_Media_Search_Handler class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since X.X.X
```

Comment:
I think it's safe to target 6.6.0 for this. It will need backporting to core like the failure on the CI suggests